topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.md

```diff
@@ -0,0 +1,597 @@
+>    - In *"Database type"*:
+>        - *"Cached database whith taxa ID"*: `NCBI-2023-06-27`
+>    - In *"Output options"*:
+>        - *"Type of extra output to be generated (default on CSV)"*: `TSV`
+>        - *"Remove the log file"*: `Yes`
+>    - In *" Fine tuning of algorithm parameters"*:
+>        - *"Strain level instead of species as the resolution limit for the robust contamination removal algorithm; use with caution, this is an experimental feature"*: `Yes`
+>
+{: .hands_on}
+</div>
+
+**Recentrifuge** generates 3 outputs:
+
+- A **statistic table** with general statistics about assignations
+
+   > <question-title></question-title>
+   >
+   > 1. How many sequences have been used?
+   > 2. How many sequences have been classified?
+   >
+   > > <solution-title></solution-title>
+   > > 1. 451,780
+   > > 2. 450,715
+   > {: .solution}
+   {: .question}
+
+- A **data table** with a report for each taxa
+
+   > <question-title></question-title>
+   >
+   > 1. How many taxa have been kept?
+   > 2. What is the lowest level in the data?
+   >
+   > > <solution-title></solution-title>
+   > > 1. 187 (190 lines including 3 header lines)
+   > > 2. The lowest level is strain.
+   > {: .solution}
+   {: .question}
+
+- A **HTML report** with a Krona chart
+
+   <iframe id="recentrifuge" src="recentrifuge_report.html" frameBorder="0" width="100%" height="900px"> ![Krona chart with multi-layered pie chart representing the community profile with in the center the higher taxonomy levels (i.e. domain) and on the exterior the more detailed ones (i.e. species)](./images/recentrifuge.png) </iframe>
+
+   > <question-title></question-title>
+   >
+   > 1. What is the percentage of classified sequences?
+   > 2. When clicking on *Staphylococcus aureus*, what can we say about the strains?
+   > 3. Is there any contamination?
+   >
+   > > <solution-title></solution-title>
+   > >
+   > > 1. 99.8%
+   > > 2. 99% of sequences assigned to *Staphylococcus aureus* are not assigned to any strains, probably because they are too similar to several strains. *Staphylococcus aureus* subsp. aureus JKD6159 is the strain with the most classified sequences, but only 0.3% of the sequences assigned to *Staphylococcus aureus*.
+   > > 3. There is no sign of a possible contamination. Most sequences are classified to taxon on the *Staphylococcus aureus* taxonomy. Only 3% of the sequences are not classified to *Staphylococcus*.
+   > >
+   > {: .solution}
+   >
+   {: .question}
+
+Once we have identified contamination, if any is present, the next step is to **remove the contaminated sequences** from the dataset to ensure the integrity of the remaining data. This can be done using bioinformatics tools designed to filter out unwanted sequences. Additionally, it's important to document and **report the contamination findings** to maintain transparency and guide any necessary adjustments in sample collection or processing protocols.
```

Comment:
Could you add a suggestion of 1-2 tools that could be used to filter the sequences?